spanner/value_test.go

```diff
@@ -476,6 +476,10 @@ func TestEncodeValue(t *testing.T) {
 		// PROTO MESSAGE AND PROTO ENUM
 		{singer1ProtoMsg, protoMessageProto(singer1ProtoMsg), tProtoMessage, "Proto Message"},
 		{singer1ProtoEnum, protoEnumProto(singer1ProtoEnum), tProtoEnum, "Proto Enum"},
+		{NullProto{singer1ProtoMsg, true}, protoMessageProto(singer1ProtoMsg), tProtoMessage, "NullProto with value"},
+		{NullProto{}, nullProto(), protoMessageType(""), "NullProto with null"},
```

Comment:
For protoMessageType(""), we are using empty string. Is it true that the string content doesn't matter? If so, what about put something like: does.not.matter? :D

Reply:
I tried it through Java Client and it doesn't matter what `protoTypeFqn` we are providing with null value it always get executed successfully.

Reply:
As mentioned in below [comment](https://github.com/googleapis/google-cloud-go/pull/6954#discussion_r1008092202), the current changes do not pass a `""` string and instead fetches the correct `protoTypeFqn` even for nil types.